exercises/F-strings-methods/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
+var name = "tony";
```

Comment:
At the moment, you're using `var` to declare a varibable. We probably need to update our exercises: however, whenever you're declaring a variable be sure to use the `let` or `const` keyword. You'll learn more about why this is the case later on in the course.